account_move_tier_validation_approver/models/tier_review.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 ForgeFlow, S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, models
+
+
+class TierReview(models.Model):
+    _inherit = "tier.review"
+
+    @api.depends("reviewer_ids")
+    def _compute_todo_by(self):
+        for review in self:
+            rec = self.env[review.model].browse(review.res_id)
```

Comment:
you should not inherit this method but use the new feature you proposed https://github.com/OCA/server-ux/pull/371